satellite/satellitedb/dbx/satellitedb.dbx

```diff
@@ -730,3 +730,38 @@ read count (
 )
 
 create user_credit ()
+
+//--- bucket metadata table ---//
+
+model bucket_metadata (
+	key project_id name
+
+	field project_id		blob
+	field name  			blob
+
+	field segment_size 		int
+
+	field path_cipher 		int
+	field enc_cipher   		int
+	field enc_blocksize		int
+
+	field rs_algorithm 		int
```

Comment:
Maybe we can drop DBX here because using `int` for all fields looks to be wasteful. @egonelbre make initial design for those fields and it looks that field types can be much smaller https://github.com/storj/storj/pull/1773/files#diff-ff21df8088ff0b306b1eb76fbbe6b85eR7

Reply:
sounds good. let me see if i can update that

Reply:
my opinion is that this is metadata per bucket and this feels like premature optimization. i vote we leave these as individual columns for now out of clarity. if we need to reclaim disk space, i think bucket metadata is the last place we'll look

Reply:
we can always do a migration if we need to

Reply:
Yeah, this was more about thinking how we will handle this for each object. Since it will be better to keep it consistent with that behavior. However, I'm fine with leaving as separate columns for now.